experimental/ubuntu/python2.7/Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Start with DBR base image, all installations for Python REPL need to be installed
+FROM databricksruntime/minimal:latest
+
+# Installs python 2.7 and virtualenv for Spark and Notebooks
+RUN apt-get update \
+  && apt-get install -y \
+    python2.7 \
+    python-dev \
+    virtualenv \
+    python-subprocess32 \
```

Comment:
is there a tldr as to why this package is needed?

Reply:
I will add one, yes!